.bluemix/deploy.json

```diff
@@ -32,6 +32,12 @@
             },
             "type": "string",
             "pattern": "\\S"
+        },
+        "metrics-tracker": {
+            "description": {
+                "$ref": "#/messages/deploy.metricsDescription"
+            },
+            "type": "string"
```

Comment:
it would be great if this were a dropdown menu like the screenshot below. I looked at the docs and couldn't find the right option though -- the screenshot was from the watson conversation pipeline config

<img width="1058" alt="screen shot 2017-11-07 at 12 00 34 am" src="https://user-images.githubusercontent.com/4118756/32478059-6a578264-c350-11e7-86b7-5dcd1ca5baab.png">

Reply:
maybe type: select? i'm not sure